packages/config/config/devices/0x0086/templates/aeotec_template.json

```diff
@@ -300,7 +300,7 @@
 			}
 		]
 	},
-	"notification_report_type": {
+	"multilevel_report_type": {
```

Comment:
Did you check that this isn't used anywhere else?

Reply:
Yes - I touched all the files from the search result https://github.com/zwave-js/node-zwave-js/search?q=notification_report_type